spec/support/active_record.rb

```diff
@@ -18,6 +18,7 @@ def define_schema!
           create_table :sensitive_data, :force => true do |t|
             t.column :name, :string
             t.column :storage, :text
+            t.column :storage_binary, :binary
```

Comment:
Are there any upgrade paths to consider for existing users? Their original table definitions may not have had this binary value, is that important?

Reply:
@jmazzi I only added this so that there was no longer a need to stub out the `columns_hash` value. If I read the test right, then [this was intended](https://github.com/jmazzi/crypt_keeper/blob/master/spec/crypt_keeper/model_spec.rb#L29) to simulate already having a binary column in the DB. Instead of doing that I just directly created one.

That means I don't think there should be a migration necessary for existing users - since their column types are already `binary`. Unless I've missed something which is entirely possible 😄